CHARTER.md

```diff
@@ -0,0 +1,138 @@
+# OKD Working Group Charter
+
+v1
+
+2019-09-13 
+
+https://github.com/openshift/community/pull/3
+
+## Introduction
+The charter describes the operations of the OKD Working Group (OKD WG).
+
+OKD is the Origin Community Distribution of Kubernetes that is upstream to Red Hat’s OpenShift Container Platform and is built around a core of OCI containers, and Kubernetes container cluster management. OKD is augmented by application lifecycle management functionality and DevOps tooling. 
+
+The OKD Working Group's purpose is to discuss, give guidance to, and enable collaboration on current development efforts for OKD4, Kubernetes and related CNCF projects. The OKD Working Group will also include the discussion of shared community goals for OKD4 and beyond. Additionally, the Working Group will produce supporting materials and best practices for end-users and will provide guidance and coordination for CNCF projects working within the SIG's scope. 
+
+The OKD Working Group is independent of both Fedora and the Cloud Native Container Foundation (CNCF). The OKD Working Group is a community sponsored by Red Hat. 
```

Comment:
```suggestion
The OKD Working Group is independent of both Fedora and the Cloud Native Container Foundation (CNCF). The OKD Working Group is a community sponsored by Red Hat.
```

Reply:
Hey, one quick note.  CNCF is Cloud Native "Computing" Foundation.
```suggestion
The OKD Working Group is independent of both Fedora and the Cloud Native Computing Foundation (CNCF). The OKD Working Group is a community sponsored by Red Hat. 
```